client/containers/DashboardOverview/OverviewBlocks.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from 'components';
+
+require('./overviewBlocks.scss');
+
+const propTypes = {
+	overviewData: PropTypes.object.isRequired,
+	useCollectionPubsObject: PropTypes.object.isRequired,
```

Comment:
Another place where these kinds of objects are sort of obscuring intent — this signature should read:

```
useCollectionPubsObject: PropTypes.shape({
    collectionPubs: PropTypes.arrayOf(PropTypes.shape({ ... })),
}).isRequired
```

`PropTypes.object` should only be used when the shape of the object is truly irrelevant to the component whose signature is being specified — e.g. it is being passed directly to a child.

Reply:
Agreed. Fixed with refactor in latest commit.